compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_block.rs

```diff
@@ -164,7 +168,6 @@ impl<'block, Registers: RegisterAllocator> BrilligBlock<'block, Registers> {
 
         if self.brillig_context.count_array_copies()
             && matches!(terminator_instruction, TerminatorInstruction::Return { .. })
-            && self.brillig_context.get_globals_memory_size().is_some()
```

Comment:
This flag should still be useful - it makes it so that we only emit this printout at the end of the current artifact, instead of at the end of every function called.

Reply:
Ah yes I see